mac/setup/build.sh

```diff
@@ -71,25 +71,28 @@ cp textinputsource/textinputsource "$TARGETAPP/Contents/MacOS/textinputsource"
 
 # NOW TO notarize...
 xattr -rc "$TARGETAPP"
-execCodeSign direct --force --options runtime --deep --sign "${CERTIFICATE_ID}" "$TARGETAPP"
 
-#
-# Notarize the app (copied from ../build.sh)
-# TODO: merge notarization into function
-#
+if ! builder_is_ci_build || ! builder_is_ci_build_level_build; then
```

Comment:
Is the reason for the compound condition to make sure we don't use the second one if the first doesn't apply?  If so, why is there not a similar check at the site of my previous comment?  (re: ...`_build_level_release`)

https://github.com/keymanapp/keyman/blob/40795d6ab7ef55e6dc8d872a057aeaf6332a7ce7/resources/builder.inc.sh#L2242-L2246

Reply:
The compound condition is needed:

1. We need to make sure that we allow run of this code on a non-CI build
2. But we don't want to run this code on a 'build' buildLevel, only on a 'release' buildLevel for CI.

This is the same as `builder_if_release_build_level`, which we could use if we refactored this further